### DISCUSSION TITLE
Add "?orgId=N" everywhere

### DISCUSSION THREAD

**Discussion by thomasbromkapsdata** [grafana/grafana#67024] [General]
Hello,
using Grafana with multiple organizations, i found very annoying that “?orgId=N” is not in every URL. By example, it appear well for dashboards but not for folders or datasources.
So it’s really easy to share a dashboard but not a folder or anything else: the link will only work well if the other user (or yourself going back to this link later) is already logged in the right organization or if you add manually the right orgId.
Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82723. Feel free to continue the discussion around this there. Thank you!
